moztelemetry/dataset.py

```diff
@@ -176,8 +178,15 @@ def records(self, sc, limit=None, sample=1, decode=None, summaries=None):
         if summaries and limit is None and sample != 1:
             if sample < 0 or sample > 1:
                 raise ValueError('sample must be between 0 and 1')
-            summaries = random.sample(summaries,
-                                      int(len(summaries) * sample))
+            # We want this sample to be reproducible.
+            # See https://bugzilla.mozilla.org/show_bug.cgi?id=1318681
+            seed_state = random.getstate()
+            try:
+                random.seed(seed)
```

Comment:
The only reason why I put it as default is because it's backward-compatible. But I don't think anybody is basing their analysis on this subtle thing. I'll change it.